ssl/quic/quic_txp.c

```diff
@@ -837,11 +837,7 @@ int ossl_quic_tx_packetiser_generate(OSSL_QUIC_TX_PACKETISER *txp,
          ++enc_level)
         txp_pkt_cleanup(&pkt[enc_level], txp);
 
-    /*
-     * If we already successfully did at least one, make sure we report this via
-     * the return code.
-     */
-    return pkts_done > 0 ? TX_PACKETISER_RES_SENT_PKT : res;
+    return res;
```

Comment:
Maybe move the comment up to 831 as well?

Reply:
I think the information from the comment is quite obvious now that we do not treat failures as OK if we sent at least one packet.